containersQa/testlib.bash

```diff
@@ -941,7 +941,7 @@ function setupAlgorithmTesting {
   checkAlgorithmsCode=`cat $LIBCQA_SCRIPT_DIR/algorithmTesting/CheckAlgorithms.java | sed -e "s/'//g"` # the ' characters are escaping and making problems, deleting them here
   cipherListCode=`cat $LIBCQA_SCRIPT_DIR/algorithmTesting/CipherList.java`
 
-  local configFileName=$(bash $LIBCQA_SCRIPT_DIR/algorithmTesting/chooseAlgorithmConfigFile.sh $OTOOL_OS $OTOOL_cryptosetup)
+  local configFileName=$(bash $LIBCQA_SCRIPT_DIR/algorithmTesting/chooseAlgorithmConfigFile.sh $OTOOL_BUILD_OS $OTOOL_cryptosetup)
```

Comment:
wait. This looks bad.

Reply:
Jdk gets its provides inruntime, not in buildtime.  So the os it is runs onis the one which metters, not the one where the jdk was built.

Reply:
Smells bad. The original one $OTOOL_OS , was correct.  I otool job algorithms should not be depnednt on build system, but only on runtime system. IN addition, our RPMs repack portables whcih so even less to distinguish. Please revert the last change, and lets merge it as it was. The need of -OTOOL_OS +OTOOL_BUILD_OS  needs separate investigations